frameworks/JavaScript/ultimate-express/src/server.mjs

```diff
@@ -0,0 +1,105 @@
+import express from "ultimate-express";
+import {
+  generateRandomNumber,
+  getQueriesCount,
+  handleError,
+  escape,
+  jsonSerializer,
+  worldObjectSerializer,
+  sortByMessage,
+  writeResponse,
+  headerTypes,
+  GREETING,
+} from "./utils.mjs";
+
+let db;
+const { DATABASE } = process.env;
+if (DATABASE) db = await import(`./database/${DATABASE}.mjs`);
+
+const extra = { id: 0, message: "Additional fortune added at request time." };
+
+const app = express();
+
+app.get("/plaintext", (req, res) => {
+  writeResponse(res, GREETING, headerTypes["plain"]);
```

Comment:
If you write it as instead
```js
app.get("/plaintext", (req, res) => {
  res.setHeader("Content-Type", "text/plain");
  res.setHeader("Server", "UltimateExpress");
  res.send("Hello, World!");
});
```
UltimateExpress will be able to compile it into declarative rule and it'll be much faster

Reply:
Thanks for letting me know. I have made the changes